lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -169,7 +178,7 @@ export default function FilePickerApp({
           You can select content for your assignment from one of the following
           sources:
         </p>
-        <input name="document_url" type="hidden" value={url} />
+        <input name="document_url" type="hidden" value={url || ''} />
```

Comment:
`url` may be `null` but an `<input>`'s value should be a string, although since `inputElement.value = null` coerces its value to an empty string both versions have the same effect.